### PR TITLE
:sparkles: :snowflake: :art: Upgrading PHP to 7.1

### PR DESCRIPTION
**Pull request by svpernova09**
* PHP 7.1
* Updated `CHANGELOG.md`
* Requires Box version 1.0.0 (PHP 7.1 box)

### CODE REVIEW

File: scripts/homestead.rb [hunk 1]
```diff
@@ -76,8 +76,8 @@ def Homestead.configure(config, settings)
     default_ports = {
       80   => 8000,
       443  => 44300,
-      3306 => 33060,
-      5432 => 54320
+      # 3306 => 33060,
```
Comment (bicatu) [inline scripts/homestead.rb]: why did you comment the port?

**Comment by svpernova09** [comment]
Nice catch! Will update. Thanks!

-
Joe Ferguson

**Comment by taylorotwell** [comment]
Ping me when updated :)

**Comment by svpernova09** [comment]
@taylorotwell This should be ready to go now. I've set the requirement of the box to be `>= 1.0.0`. This version hasn't been released yet. I'm still waiting for the VMWare box to upload to Atlas.

I'll tag a new Homestead release (3.2.0) once the box is finished uploading and released on Atlas.

**Pull request merged**